### PR TITLE
Tests: add connectivity test between two containers

### PR DESCRIPTION
**Pull request by chavafg**
This test runs a simple ping from one Container to another
to verify connectivity.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: tests/integration/docker/network.bats [hunk 1]
```diff
@@ -30,4 +30,11 @@ setup() {
 @test "HostName is passed to the container" {
 	hostName=clr-container
 	$DOCKER_EXE run -h $hostName -i ubuntu hostname | grep $hostName
-}
\ No newline at end of file
+}
+
+@test "Verify connectivity between 2 containers" {
+	contName='pingTest'
+	docker run -tid --name "$contName" ubuntu bash
```
Comment (jcvenegas) [inline tests/integration/docker/network.bats]: other tests use DOCKER_EXE variable

Comment (chavafg) [inline tests/integration/docker/network.bats]: good catch, fixed.

**Comment by jcvenegas** [comment]
+1

**Comment by devimc** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/01org/cc-oci-runtime/pull-request/210/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=01org/cc-oci-runtime)

**Pull request merged**